src/doodle/canvas.js

```diff
@@ -1,5 +1,6 @@
 import { createElement } from 'preact';
-import {useRef, useEffect} from 'preact/hooks';
+import { useRef, useEffect } from 'preact/hooks';
+import propTypes from 'prop-types';
 
 //TODO not sure how to enforce this type...
```

Comment:
Types are now enforced, so we can remove this TODO right? Or was this in reference to local variables?